deploy/olm-catalog/mobile-developer-console-operator/0.3.0/mobile-developer-console-operator.v0.3.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,270 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: '[{"apiVersion":"mdc.aerogear.org/v1alpha1","kind":"MobileDeveloperConsole","metadata":{"name":"example-mdc"},"spec":{"oAuthClientId":"mobile-developer-console","oAuthClientSecret":"foobar"}}]'
```

Comment:
are we okay with the oAuthClientSecret being foobar here?

Reply:
I don't think what's in here really matters, but I could be wrong. Is there something you'd suggest changing it to?

Reply:
Yeah, I'm not really sure. I think it's fine for now anyway.

Reply:
I don't think it matters much and from the point of view of the integreatly operator it gets replaced by the openshift oauth client